index.html

```diff
@@ -11,9 +11,23 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+<script src="script.js" defer></script>
+<script src="contributors.js" defer></script>
+<link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
-    <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+<body id="about-page">
+    <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+        <div id="pop-up-menu-wrapper">
+            <i class="fa-solid fa-xmark" id="cancel"></i>
+        <div class="pop-up-menu">
+            <ul class=".pop-up-list">
+                <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
```

Comment:
- [ ] In your home page in the desktop version i would point some things in your main progtam section I think it would be very nice to have some animation to your cards as shown in the given design so whenever the user press on the card so it make a border to the card like the given design and that's important to improve from user experience (UX) ♻⭐💯🚀🔥

- [ ] Great job so far but I think it would be great if you add more gaps between to the cards to match the original desgin as much as possible so please add more height to the cards here 🔥🙏🏻
- Your design
![image](https://user-images.githubusercontent.com/58553711/219921571-8f84c212-93aa-4dbb-ac8b-fe833cd91ff5.png)


-Given 
![image](https://user-images.githubusercontent.com/58553711/219341778-278670b7-c342-450a-9d35-6e39a12c0785.png)